src/_includes/server/http/CGIClass.hpp

```diff
@@ -13,12 +13,13 @@
 
 namespace NotApache {
 
-#define EXECVE_ERROR	1
-#define CLOSE_ERROR		2
-#define DUP2_ERROR		3
-#define CHDIR_ERROR		4
-#define GETCWD_ERROR	5
-#define MEMORY_ERROR	6
+	// defines start are 128 because they arent reserved exit codes
+	#define EXECVE_ERROR	129
+	#define CLOSE_ERROR		130
+	#define DUP2_ERROR		131
+	#define CHDIR_ERROR		132
+	#define GETCWD_ERROR	133
+	#define MEMORY_ERROR	134
```

Comment:
Technically these are signals codes, but we don't add 128 to the exit status so it should be fine

Reply:
I couldnt find any info on signal codes by googling. care to elaborate?

Only thing I know is that these exit codes are used by shells to indicate certain shell specific errors. since we arent running shells they should be fine to use. I encountered an issue where an abort is exit code 6. so it would give a invalid error message

Reply:
Yes mostly for shells, it could be confusing if the program is run with `make`. For example with an abort, make will return 134 instead of 6. But our defines are used internally, so it shouldn't be a problem.